crates/pg_lsp_new/src/handlers/text_document.rs

```diff
@@ -35,9 +35,9 @@ pub(crate) async fn did_open(
 
     session.insert_document(url.clone(), doc);
 
-    // if let Err(err) = session.update_diagnostics(url).await {
-    //     error!("Failed to update diagnostics: {}", err);
-    // }
+    if let Err(err) = session.update_diagnostics(url).await {
+        error!("Failed to update diagnostics: {}", err);
+    }
```

Comment:
Same here, should we inform the client as well?

Reply:
just for debugging